content/documentation/pages/1-installation/1-local/1-docker.md

```diff
@@ -0,0 +1,134 @@
+
+The `docker-compose.yml` file defines `DATAFLOW_VERSION` and `SKIPPER_VERSION` variables, so that those values can
+be easily changed.
+The preceding commands first set the `DATAFLOW_VERSION`, `SKIPPER_VERSION` to use and then `docker-compose` is started.
+
+You can also use a shorthand version that exposes only the `DATAFLOW_VERSION`, `SKIPPER_VERSION` variables to the `docker-compose` process (rather than setting it in the environment), as follows:
+
+```
+DATAFLOW_VERSION={local-server-image-tag} SKIPPER_VERSION={skipper-version} docker-compose up
+```
+
+[[note]]
+| By default, the latest GA releases of Stream and Task applications are imported automatically.
+
+[[tip]]
+| By default, Docker Compose uses locally available images. For example, when using the `latest` tag, run `docker-compose pull` prior to
+| `docker-compose up` to ensure the latest image is downloaded.
+
+[[tip]]
+| Some stream applications may open a port, for example `http --server.port=`. By default, a port range of `9000-9010` is exposed from the
+| container to the host. If you would need to change this range, you can modify the `ports` block of the `dataflow-server` service in
+| the`docker-compose.yml` file.
+
+Now that docker compose is up, you can access the Spring Cloud Data Flow Dashboard.
+In your browser, navigate to the [Spring Cloud Data Flow Dashboard URL](http://localhost:9393/dashboard).
+
+## Stop Docker Compose
+
+When you want to shut down Data Flow use the `docker-compose down` command.
+
+1.  Open a new terminal window.
+
+2.  Change directory to the directory in which you started (where the
+    `docker-compose.yml` file is located).
+
+3.  Run the following command:
+
+        $ DATAFLOW_VERSION={local-server-image-tag} SKIPPER_VERSION={skipper-version} docker-compose down
+
+<!--NOTE-->
+
+You need to specify the `DATAFLOW_VERSION` and the `SKIPPER_VERSION` environment variables if you are running the command in a separate terminal window from which you started Docker Compose. The `export` commands you used earlier set the variables for only that terminal window, so those values are not found in the new terminal window.
+
+<!--END_NOTE-->
+
+If all else fails, you can shut down Docker Compose using `Ctrl+C`.
+
+## Shell
+
+For convenience and as an alternative to using the Spring Cloud Data Flow Dashboard, the Spring Cloud Data Flow Shell can be used as an alternative to the UI.
+The shell supports tab completion for commands and also for stream/task application configuration properties.
+
+If you have started Data Flow using Docker compose, the shell is is also included in the springcloud/spring-cloud-dataflow-server Docker image.
+To use it, open another console window and type the following:
+
+    $ docker exec -it dataflow-server java -jar shell.jar
+
+If you have started the Data Flow server via `java -jar`, download and start the shell using the following commands:
```

Comment:
This line needs to be replaced since we are only discussing docker-compose deployment.   It could be replaced with something like, "You can run the Spring Cloud Data Flow shell without docker using the following instructions:"

Reply:
Moved into the 3-manual section.